modules/rhel-compute-requirements.adoc

```diff
@@ -19,7 +19,7 @@ ifdef::openshift-origin[]
 ** Base OS: CentOS 7.4.
 endif::[]
 ifdef::openshift-enterprise,openshift-webscale[]
-** Base OS: link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_a_standard_rhel_installation/index[{op-system-base} 8.4 or 8.5] with "Minimal" installation option.
+** Base OS: link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_a_standard_rhel_installation/index[{op-system-base} 8.5 or 8.6] with "Minimal" installation option.
```

Comment:
```suggestion
** Base OS: link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_a_standard_rhel_installation/index[{op-system-base} 8.5 or later] with "Minimal" installation option.
```

Reply:
For 4.11 this should be RHEL8 version 8.5 or later, for 4.10 this should be RHEL8 version 8.4 or later, and for 4.9 this should be RHEL7 version 7.9 or RHEL8 version 8.4 or later.

The goal is to convey that we set a minimum minor version and as long as it's greater than that but not RHEL9 we will support it.

Reply:
After discussing on Slack, this is more simply written as "8.6-8.7" which will be accurate for 4.12.